test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,23 @@
 
 public class UtilsTest {
 
+    @Test
+    public void elementsContainNull() throws Exception {
+        //empty list
+        assertFalse(Utils.isAnyNull());
+
+        //only 1 object
+        assertTrue(Utils.isAnyNull((Object) null));
+        assertFalse(Utils.isAnyNull(1));
+        assertFalse(Utils.isAnyNull(""));
+        assertFalse(Utils.isAnyNull("abc"));
+
+        //some nulls
+        assertTrue(Utils.isAnyNull((Object) null, (Object) null));
+        assertTrue(Utils.isAnyNull((Object) null, 1));
+        assertTrue(Utils.isAnyNull("abc", (Object) null));
+        assertFalse(Utils.isAnyNull("abc", "abc"));
+    }
```

Comment:
An interesting test case that you can also add would be to check for nulls inside lists. For example:

```
List<Object> nullList = Arrays.asList((Object) null);
assertFalse(Utils.isAnyNull(nullList));
```